packages/server/src/core/internals/procedureBuilder.ts

```diff
@@ -44,12 +42,43 @@ type CreateProcedureReturnInput<
   _output_out: FallbackValue<TNext['_output_out'], TPrev['_output_out']>;
 }>;
 
+/**
+ * @internal
+ */
+export interface BuildProcedure<
```

Comment:
This doesn't actually help but I did it and it makes our source code more concise anyway :shrug: